core/src/validator.rs

```diff
@@ -158,6 +158,7 @@ pub struct ValidatorConfig {
     pub validator_exit: Arc<RwLock<Exit>>,
     pub no_wait_for_vote_to_start_leader: bool,
     pub accounts_shrink_ratio: AccountShrinkThreshold,
+    pub no_hard_fork_blockstore_root_reconcilation_for_local_cluster_test: bool,
```

Comment:
Maybe a comment on why this is necessary?

Reply:
this got n/a because i've simplified the tests.